tests/bluetooth/tester/prj.conf

```diff
@@ -47,6 +47,8 @@ CONFIG_BT_ISO_BROADCASTER=y
 CONFIG_BT_ISO_SYNC_RECEIVER=y
 CONFIG_BT_ISO_CENTRAL=y
 CONFIG_BT_ISO_PERIPHERAL=y
+CONFIG_BT_ISO_RX_BUF_COUNT=4
+CONFIG_BT_ISO_TX_BUF_COUNT=4
```

Comment:
those are already set to those value in overlay-le-audio.conf
I'd suggest to remove them from there if already set here

Reply:
yes, but prj.conf is always included so no need to have those also in leaudio overlay